corehq/apps/users/tasks.py

```diff
@@ -341,3 +343,30 @@ def process_reporting_metadata_staging():
     run_again = run_periodic_task_again(process_reporting_metadata_staging_schedule, start, duration)
     if run_again and UserReportingMetadataStaging.objects.exists():
         process_reporting_metadata_staging.delay()
+
+
+@periodic_task(run_every=crontab(minute='*/10'), queue='background_queue')
```

Comment:
`minute='*/10'` means this runs every 10 minutes, right? While looking into [USH-912](https://dimagi-dev.atlassian.net/browse/USH-912) I noticed this task shows up a lot in the [long-running tasks graph](https://app.datadoghq.com/dashboard/bdu-k5b-bz5/celery-overview?from_ts=1617666208741&fullscreen_end_ts=1618271078052&fullscreen_paused=false&fullscreen_section=overview&fullscreen_start_ts=1617666278052&fullscreen_widget=2868802710810996&live=true&to_ts=1618271008741&tpl_var_celery_queue=background_queue&tpl_var_celery_task=%2A), in the 10-60 minutes bucket. I don't have context on this task, but curious how important it is that this run so frequently?